package.json

```diff
@@ -38,6 +38,7 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-redux": "^5.0.7",
+    "react-router-dom": "^4.3.1",
```

Comment:
I though that `react-router` also needed to be moved to `devDependencies`?

Reply:
added it according to this doc: https://github.com/folio-org/stripes/blob/stripes-v3/doc/upgrade-v3.md